eng/pipelines/common/xplat-setup.yml

```diff
@@ -30,6 +30,21 @@ jobs:
       - name: System.DisableZipDownload
         value: true
 
+      - ${{ if eq(parameters.jobParameters.buildConfig, 'checked') }}:
+        - name: buildConfigUpper
+          value: 'Checked'
+
+      - ${{ if eq(parameters.jobParameters.buildConfig, 'debug') }}:
+        - name: buildConfigUpper
+          value: 'Debug'
+
+      - ${{ if eq(parameters.jobParameters.buildConfig, 'release') }}:
+        - name: buildConfigUpper
+          value: 'Release'
+
+      - name: _BuildConfig
+        value: $(buildConfigUpper)
```

Comment:
What's the reason that we are uppercasing the build config? Our entrypoints allow lowercase configurations.

Reply:
This is for the artifacts and for the difference in between coreclr, libraries and installer. CoreCLR uses lowercase but the artifacts need to be on upper-case etc. I think to try to fix this on this PR is out of the scope here, but something we should definitely fix.

Reply:
Can you please open an issue? Unsure if I have all the context.

Reply:
Yeah. I unresolved the conversation to remember.